src/bandersnatch/filter.py

```diff
@@ -29,6 +33,7 @@ class Filter:
     """
 
     name = "filter"
+    deprecated_name: Optional[str] = None
```

Comment:
```suggestion
    deprecated_name: str = “”
```
NIT Pick: I see no benefit of non versus a no truths (false) empty str
- won’t block on it tho

Reply:
No worries, have updated this.